Mathlib/Data/Set/Pointwise/Interval.lean

```diff
@@ -27,6 +27,93 @@ variable {α : Type*}
 
 namespace Set
 
+/-! ### Binary pointwise operations
+
+Note that the subset operations below only cover the cases with the largest possible intervals on
+the LHS: to conclude that `Ioo a b * Ioo c d ⊆ Ioo (a * c) (c * d)`, you can use monotonicity of `*`
+and `Set.Ico_mul_Ioc_subset`.
```

Comment:
Can we have a TODO for the analogous lemmas in the generality of `mul_le_mul` rather than `mul_le_mul'`? I need these for the exponential ramsey project, so I don't mind PRing them myself later. But of course feel free to add them here anyway.

Reply:
Done. I moved all my existing lemmas to occupy the primed names.

Reply:
I think you should unprime them and have `\_0` or `_of_pos`/`_of_nonneg` for the other ones instead. I'm trying to deprecate priming ordered monoid lemmas (it leads to a confusing mess of a naming convention) and I would like there not to be more lemma names to fix.

Reply:
I'd rather optimize for current consistency rather than eventual consistency that may be never be reached.

Reply:
Well yeah, if you work against it...

Reply:
This is a very real thing that is going to happen really soon, btw. I have a gitpod with 150 files changed in it to fix one collection of lemma names, and `mul_le_mul'` will be my next batch.

Reply:
At this point me reverting the names now is no harder than you doing so in your chain of PRs, and thematically the de-priming makes much more sense there.